pkgs/tools/graphics/realcugan-ncnn-vulkan/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, vulkan-headers
+, vulkan-loader
+, glslang
+, libgcc
+, libwebp
+, ncnn
+}:
+
+stdenv.mkDerivation rec {
+  pname = "realcugan-ncnn-vulkan";
+  version = "20220318";
+
+  src = fetchFromGitHub {
+    owner = "nihui";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-pmh73rBRLRpHy0uNgiAM9Y04GGEhEbR5M13oTf1kOYI=";
+  };
+  sourceRoot = "source/src";
+
+  patches = [
+    # Fixes missing target SPIRV-Tools-opt
+    ./cmakelists.patch
+  ];
+
+  cmakeFlags = [
+    "-DUSE_SYSTEM_NCNN=1"
+    "-DUSE_SYSTEM_WEBP=1"
+    "-DGLSLANG_TARGET_DIR=${glslang}/lib/cmake"
+  ];
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ vulkan-headers vulkan-loader glslang libwebp ncnn ]
+    ++ lib.optional (!stdenv.isDarwin) libgcc;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin $out/opt
+    install -Dm555 realcugan-ncnn-vulkan $out/bin/
+    cp -r ${src}/models/models-se $out/opt
+    cp -r ${src}/models/models-nose $out/opt
+    ln -s $out/opt/models-se $out/bin
+    ln -s $out/opt/models-nose $out/bin
```

Comment:
```suggestion
    install -Dm755 realcugan-ncnn-vulkan $out/opt
    ln -s $out/opt/realcugan-ncnn-vulkan $out/bin
    cp -r ./models $out/share
    ln -s $out/share/models/* $out/opt
```

Reply:
Does this need to add `$out/share` to the `mkdir` on line 42?